SampleApp/SampleApp/SampleApp.csproj

```diff
@@ -16,5 +16,7 @@
     <SharedImage Include="loginbg.png" BaseSize="123,105" />
 
     <SharedImage Include="appicon.svg" IsAppIcon="true" ForegroundFile="appiconfg.svg" ForegroundScale="0.65" />
+    <SharedImage Include="camera.svg" IsAppIcon="true" ForegroundFile="appiconfg.svg" ForegroundScale="0.65" />
```

Comment:
@mattleibow @Redth what happens when you set two icons to have IsAppIcon to true? Does it end up using the 2nd one?

Reply:
Ah I see! So you basically use something like [this on Android](https://www.geeksforgeeks.org/how-to-change-app-icon-of-android-programmatically-in-android/) and [this on iOS](https://prototypemakers.medium.com/userscustomizeiosappiconsxamarin-9d08b671e4e8)